backend/src/main/java/com/wpz/rbs/service/RoomService.java

```diff
@@ -0,0 +1,44 @@
+package com.wpz.rbs.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.wpz.rbs.model.Room;
+import com.wpz.rbs.repository.RoomRepository;
+
+@Service
+public class RoomService {
+    
+    @Autowired
+    RoomRepository roomRepository;
+
+    public List<Room> getAll(){  
+        List<Room> rooms = new ArrayList<Room>();  
+        roomRepository.findAll().forEach(room -> rooms.add(room));  
+        return rooms;  
+    }  
+ 
+    public Room getById(int id){  
+        return roomRepository.findById(id).get();  
+    }  
+
+    // When updating, keeps the "adnotation" member, unless it's empty ("")
+    public void saveOrUpdate(Room example){  
+        var optional = roomRepository.findById(example.getId());
+        if(optional.isPresent()){
+            var existingAdnotation = optional.get().getAdnotation();
+
+            if(existingAdnotation.equals("")){
```

Comment:
when we will be want to update our annotaton this part will won't work

Reply:
We present the default state, the empty string, as valid. With that in mind, after setting the annotation, it should be possible to return to such state, that being, the empty string. Just updating the annotation is going to require another function to be added, but topic as a whole should be discussed.